src/api/app/models/label_global.rb

```diff
@@ -0,0 +1,54 @@
+# TODO: Please overwrite this comment with something explaining the model target
+class LabelGlobal < ApplicationRecord
```

Comment:
@eduardoj I'm not so sure about the naming. Those are are labels for projects. Wouldn't it make sense to name it as such? Also the `Label` model has an polymorphic association, right now only used for `Package`. Couldn't we reuse it also for `Project`?

Reply:
discussed offline